src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -62,6 +115,20 @@ def __init__(self):
         self.fail_open = False
         self.insecure_mode = False
 
+    def set_event_sub_type(self, event_sub_type):
```

Comment:
I'd like for everyone to start adding type hints to new functions.
Let me help adding them here, but for more info you could read: https://docs.python.org/3/library/typing.html
You could just accept my suggestions and then you don't need to add these locally first.

```suggestion
    def set_event_sub_type(self, event_sub_type: str) -> None:
```

Reply:
Thanks for the pointer. I know what type hints are :) , will remember to add them.